google/cloud/bigquery/v2/minimal/mocks/mock_job_connection.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_V2_MINIMAL_MOCKS_MOCK_JOB_CONNECTION_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGQUERY_V2_MINIMAL_MOCKS_MOCK_JOB_CONNECTION_H
+
+#include "google/cloud/bigquery/v2/minimal/internal/job_connection.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace bigquery_v2_minimal_mocks {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+// Class to mock `BigQueryJobConnection`.
```

Comment:
I would copy this comment chunk: https://github.com/googleapis/google-cloud-cpp/blob/f81ff8b2b295b40754b84302006732621857b300/google/cloud/accessapproval/v1/mocks/mock_access_approval_connection.h#L30-L38

(but not the `@see` note, because that applies more to gRPC mocking.)